server/views/pages/explore.njk

```diff
@@ -28,6 +28,22 @@
     </div>
   </div>
 
+  <div class="row {{ {s:2, m:2, l:2} | spacingClasses({padding:['top']}) }}">
+    <div class="container container--scroll-xl">
+      <div class="grid grid--no-gutters">
+        <div class="{{ {s: 4, m: 8, l: 12, xl: 12} | gridClasses }}">
+          {% set asyncContentEndpoint = '/series-transporter/' + latestDigitalStory %}
+          {% component 'async', { endpoint: asyncContentEndpoint, component: 'series-transporter' } %}
```

Comment:
Do you mean titlecase and non-hyphenated? This ('series-transporter') isn't what's being sent to Mixpanel – it is 'Transporter', as determined [in the numbered-list](https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/views/components/numbered-list/index.njk#L15)